src/components/dar_collection_table/Actions.jsx

```diff
@@ -221,6 +240,8 @@ export default function Actions(props) {
       {actions.includes('Cancel') && <TableIconButton {...cancelButtonAttributes} />}
       {checkEnv(envGroups.NON_PROD) && actions.includes('Create_Progress_Report') &&
         <SimpleButton {...createProgressReportButtonAttributes} />}
+      {checkEnv(envGroups.NON_PROD) && actions.includes('Review_Progress_Report') &&
```

Comment:
Do we have a ticket to remove this check yet?

Reply:
I'm not sure, and you're right, we'll need to remove these.